src/resources/language/_language-de.yml

```diff
@@ -64,13 +64,13 @@ crossref-lof-title: "Abbildungsverzeichnis"
 crossref-lot-title: "Tabellenverzeichnis"
 crossref-lol-title: "Listingverzeichnis"
 
-environment-proof-title: "Nachweisen"
-environment-remark-title: "Bemerkung"
+environment-proof-title: "Beweis"
+environment-remark-title: "Anmerkung"
 environment-solution-title: "Lösung"
```

Comment:
I would keep "Bemerkung"

Reply:
Okay I made that change. @skeydan and @mchlbckr if you continue discussing I'll track the discussion and change to whatever consensus emerges from the discussion.